src/components/inversionSolution/InversionSolutionSpectralAccelerationTab.tsx

```diff
@@ -0,0 +1,143 @@
+import { XY } from '../../interfaces/common';
+import { curveLinear } from '@visx/curve';
+import { Tooltip } from '@visx/xychart';
+import SelectControl from '../common/SelectControl';
+import { toProperCase } from '../../utils';
+
+interface InversionSolutionSpectralAccelerationTabProps {
+  id: string;
+}
+
+const InversionSolutionSpectralAccelerationTab: React.FC<InversionSolutionSpectralAccelerationTabProps> = ({
+  id,
+}: InversionSolutionSpectralAccelerationTabProps) => {
+  const data = useLazyLoadQuery<InversionSolutionSpectralAccelerationTabQuery>(
+    inversionSolutionSpectralAccelerationTabQuery,
+    { id },
+  );
+
+  const options = getHazardTableOptions(data);
+
+  const [location, setLocation] = useState<string>(options.location[0]);
+  const [forecastTime, setForecastTime] = useState<string>(options.forecastTime[0]);
+  const [gmpe, setGmpe] = useState<string>(options.gmpe[0]);
+  const [backgroundSeismicity, setBackgroundSeismicity] = useState<string>(options.backgroundSeismicity[0]);
+  const [POE, setPOE] = useState<string>('2%');
+
+  const [dataSet, setDataSet] = useState<XY[]>([]);
+
+  useEffect(() => {
+    const xValue = POE === '2%' ? 1 / 2475 : 1 / 475;
+    const filteredCurves = filterMultipleCurves(options.PGA, data, location, forecastTime, gmpe, backgroundSeismicity);
+    const plotData = getSpectralAccelerationData(options.PGA, xValue, filteredCurves);
+    setDataSet(plotData);
+  }, [location, forecastTime, gmpe, backgroundSeismicity, POE]);
+
+  return (
+    <>
+      <SelectControl name="Location" options={options.location} setOptions={setLocation} />
+      <SelectControl name="Forecast Timespan" options={options.forecastTime} setOptions={setForecastTime} />
+      <SelectControl
+        name="Background Seismicity"
+        options={options.backgroundSeismicity}
+        setOptions={setBackgroundSeismicity}
+      />
+      <SelectControl name="Background Motion Model" options={options.gmpe} setOptions={setGmpe} />
+      <SelectControl name="Probability of Exceedence" options={['2%', '10%']} setOptions={setPOE} />
+      <Box>
+        <Card>
+          <Box style={{ width: '100%', padding: '1rem' }}>
+            <div style={{ position: 'relative', width: '100%' }}>
+              <XYChart
+                height={700}
+                width={900}
+                xScale={{ type: 'linear', domain: [-1, 10] }}
+                yScale={{ type: 'linear', domain: [0, 6] }}
+              >
+                <text
+                  y={23}
+                  x={20}
+                  fontSize={20}
+                  fontWeight="bold"
+                >{`${location} Uniform Hazard Spectrum (opensha)`}</text>
+                <text y={42} x={20} fontSize={15}>
+                  {` Model: ${gmpe}. Background: ${toProperCase(
+                    backgroundSeismicity,
+                  )}d. Forecast: ${forecastTime} years.  POE: ${POE}`}
```

Comment:
Is there a reason we don't use the `label` property on the AnimatedAxis?

Reply:
True I missed that you could do it in an easier way, will refactor.